continuous_integration/circle_ci_test_doc.sh

```diff
@@ -0,0 +1,10 @@
+#!bin/bash
+
+# on circle ci, each command run with it's own execution context so we have to
+# activate the conda testenv on a per command basis. That's why we put calls to
+# python (conda) in a dedicated bash script and we activate the conda testenv
+# here.
+source activate testenv
+
+# pipefail is necessary to propagate exit codes
+set -o pipefail && cd doc && make html-strict 2>&1 | tee ~/log.txt
```

Comment:
Use `make html` here if you don't want to treat warnings as errors.

In general warnings indicates that there is a problem with your ReST so that may mean that the formatting will be a bit weird.

I'll try to take a look at your warnings although I probably get around to it before Monday.